spec/features/hq/create_sponsorship_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.feature 'Create a sponsorship', :type => :feature do
+
+  background do
+    i_sign_in_as_admin
+    a_sponsor_exists "First Sponsor"
+    an_orphan_exists "First Orphan"
+    an_orphan_exists "Second Orphan"
+  end
+
+  scenario 'Pairing a sponsor with orphans' do
+    when_i_visit_show_sponsor "First Sponsor"
+    and_i_should_be_on "hq_new_sponsorship_path", "First Sponsor"
+    and_i_should_see "First Sponsor"
+    and_i_should_see "First Orphan"
+    and_i_should_see "Second Orphan"
+    fill_orphan_with "sponsorship_start_date", "First Orphan", "2014-01-01"
+    when_i_click "Sponsor this orphan", "First Orphan"
+    and_i_should_be_on "hq_new_sponsorship_path", "First Sponsor"
+    and_i_should_see "Return to Sponsor Page"
+    and_i_should_see "No (1/2)"
+    and_i_should_see "Sponsorship link was successfully created for First Orphan"
+    fill_orphan_with "sponsorship_start_date", "Second Orphan", "2014-02-01"
+    when_i_click "Sponsor this orphan", "Second Orphan"
+    and_i_should_be_on "hq_sponsor_path", "First Sponsor"
+    and_i_should_see_within "Currently Sponsored Orphans", "First Orphan"
```

Comment:
add: `and_i_should_see "Sponsorship link was successfully created"`
to also test the presence of the flash message